app/views/teams/memberships/index.html.haml

```diff
@@ -9,9 +9,16 @@
         %table
           -@invitations.each do |invitation|
             %tr
-              %td= invitation.email
+              %td
+                = invitation.email
               -if current_team_membership.admin?
-                %td= link_to "Resend invite", "#"
+                %td
+                  %ul.inline-list
+                    %li= link_to "Resend invite", "#"
+                    %li= link_to "Remove",
+                      teams_invitation_path(invitation.id),
```

Comment:
See comment in routes.rb. And then this can change to `teams_team_invitation_path(@team.id, invitation.id)`